server/conf/cms/secrets.sample.py

```diff
@@ -14,8 +14,7 @@
 _ALLOWED_HOSTS = ['0.0.0.0', '127.0.0.1', 'localhost', '*']   # In development.
 
 # Boolean check to see if ldap is being used by the site.
-# Ensure the django-auth-ldap==2.0.0 package is uncommented
-# in the requirements.txt file or installed if using ldap.
```

Comment:
This is not necessary anymore. It is enabled dynamically based on this setting.

Reply:
Hm… actually. I lie. Dynamic nothing. This is removed from CMS, but under a false assumption.

Reply:
Task created to resolve this on CMS's secrets (then propagate to Portal's CMS secrets): https://github.com/TACC/Core-CMS/issues/179

Reply:
To be fair, we could just *always* install the `django-auth-ldap==2.0.0` package (via `requirements.txt`) regardless of the LDAP capability being enabled. 

That would be one less moving part in a fresh deployment and I don't see any overt risk coming from an unused ldap package installed in a SAD CMS site which is the current/only use case for `_LDAP_ENABLED = False`.

Reply:
That seems like it would be currently be the case for any recent CMS (unless `requirements.txt` is maintained manually on servers). Nothing has blown up. I am fine either way. Requirements of TACC/Core-CMS#179 can be satisfied with just a comment clarifying that, "Yes, we do this. No harm."

P.S. I don't know how to dynamically add a requirement based on `secrets.py`.

Reply:
I don't think there is a way to dynamically effect requirements from settings, so this seems a No Harm setup.